person.rb

```diff
@@ -1,5 +1,5 @@
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] Please ensure that your `:rentals` uses `attr_accessor` also. :slightly_smiling_face: 
- [ ] Please ensure that your `Person` class has `A method for adding rentals`